dhis-2/dhis-test-web-api/src/test/java/org/hisp/dhis/webapi/controller/tracker/imports/TrackerImportControllerTest.java

```diff
@@ -61,6 +62,22 @@ void shouldSucceedWhenAllValidParametersArePassed() {
             .content(HttpStatus.OK));
   }
 
+  @Test
+  void shouldReturnBadRequestWhenThereIsAnInvalidUidInThePayload() {
```

Comment:
Why the split of tests into e2e and web-api? Only using one would be less surprising unless there is a good reason. I prefer the web-api ones.

Reply:
Yeah, this split was weird.
I will remove the json test here and just have the web-api one but I will leave the csv as an e2e, we don't have, at least now, an easy way to test that in the web-api.